api/build.gradle

```diff
@@ -83,7 +83,7 @@ dependencies {
         exclude(module: 'commons-logging')
         exclude(module: 'slf4j-simple')
     }
-    implementation group: 'org.liquibase', name: 'liquibase-core', version: '4.23.1'
+    implementation group: 'org.liquibase', name: 'liquibase-core'
```

Comment:
Something else is forcing the version down, and is incompatible with the latest `snakeyaml`, therefore moved the resolution into `cve-resolution-strategy` below.

Reply:
chi chi